src/iiiftilesource.js

```diff
@@ -444,7 +444,7 @@ $.extend( $.IIIFTileSource.prototype, $.TileSource.prototype, /** @lends OpenSea
     /**
      * Determine whether arbitrary tile requests can be made against a service with the given profile
      * @function
-     * @param {array} profile - IIIF profile array
+     * @param {array} options - IIIF profile array
```

Comment:
Looks like this isn't quite right either. If you look at the code, it's more like this:

```
     * @param {Object} options
     * @param {Array|String} options.profile
     * @param {Number} options.version
     * @param {Number} options.extraFeatures
```

I don't know enough about IIIF to know what all these parameters mean. If you do, please add some additional commentary if appropriate.

Also, I don't think this is true:

```
 * @throws {Error}
```

… At least I don't see anywhere where it would be doing that.

It does return a Boolean, though, so it might be worth adding:

```
* @return {Boolean}
```